vespa/application.py

```diff
@@ -1651,9 +1651,9 @@ async def feed_data_point(
         schema: str,
         data_id: str,
         fields: Dict,
-        namespace: str = None,
-        groupname: str = None,
-        semaphore: asyncio.Semaphore = None,
+        namespace: str | None = None,
+        groupname: str | None = None,
+        semaphore: asyncio.Semaphore | None = None,
```

Comment:
```suggestion
        namespace: Optional[str] = None,
        groupname: Optional[str] = None,
        semaphore: Optional[asyncio.Semaphore] = None,
```

Reply:
This syntax was [introduced in Python 3.10](https://peps.python.org/pep-0604/), so we can't switch to it yet unfortunately.